### DISCUSSION TITLE
Commands fabric-ca-client

### DISCUSSION THREAD

**Discussion by douglasjava** [hyperledger/fabric-ca#362] [Q&A]
Good evening, does this fabric-ca-client CLI have documentation? with it is possible to obtain information from the network and manage it?

error by try that command above

![image](https://github.com/hyperledger/fabric-ca/assets/20793718/80ad5426-0b0a-41fa-9a06-f4c1b85c64a7)

**Comment by bestbeforetoday** [comment]
The [README](https://github.com/hyperledger/fabric-ca#readme) for this repository contains a link to the [Fabric CA User Guide](https://hyperledger-fabric-ca.readthedocs.io/), and that includes documentation for the [fabric-ca-client CLI](https://hyperledger-fabric-ca.readthedocs.io/en/latest/users-guide.html#fabric-ca-client).

Your screenshot does not show the `fabric-ca-client` command being used. Instead, it shows the Fabric `peer` command being used. For documentation of the Fabric `peer` command, I suggest you consult the Commands Reference in the main [Fabric documentation](https://hyperledger-fabric.readthedocs.io/).
